WebUI/src/components/AddLLMDialog.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div class="dialog-container z-10">
+        <div class="dialog-mask absolute left-0 top-0 w-full h-full bg-black/55 flex justify-center items-center">
+            <div class="py-10 px-20 w-500px flex flex-col items-center justify-center bg-gray-600 rounded-3xl gap-6 text-white"
+                :class="{ 'animate-scale-in': animate }">
+              <p v-html= "i18nState.REQUEST_LLM_MODEL_NAME"></p>
+              <textarea class="rounded-xl border border-color-spilter flex-auto w-full h-auto resize-none" rows="1"
+                        :placeholder="languages.COM_LLM_HF_PROMPT" v-model="modelRequest" @keydown="fastGenerate"></textarea>
+              <p v-show = "addModelError" style="color: #F44336;">{{ addModelErrorMessage }}</p>
```

Comment:
any reason to not just select another tailwind utility class for the color style?

Reply:
tailwind css reds did not fit the background color well.